utilities/src/main/java/io/syndesis/qe/resource/impl/Syndesis.java

```diff
@@ -163,7 +163,7 @@ public void createPullSecret() {
     public void jaegerWorkarounds() {
         new Thread(() -> {
             try {
-                OpenShiftWaitUtils.waitUntilPodAppears("jaeger-operator");
+                OpenShiftWaitUtils.waitUntilPodAppearsOrAnotherPodPresent("jaeger-operator", "syndesis-jaeger");
```

Comment:
maybe it would be better to change `waitUntilPodAppears(String podName)` into `waitUntilAnyPodAppears(String... podNames)`, wdyt?

Reply:
Yes, it would be probably better :D